scss/modules/_our-horses.scss

```diff
@@ -0,0 +1,218 @@
+@use "./common/forwarded" as helpers;
+
+.our-horses {
+
+	&__container {
+		max-width: 100%;
+		margin: 0 auto 70px;
+		overflow: hidden;
+		padding: 0 10px;
+
+		@include helpers.mobile() {
+			margin: 0 auto 61px;
+		}
+
+		@include helpers.tablet() {
+			padding: 0 20px;
+			margin: 0 auto 80px;
+		}
+
+		@include helpers.laptop() {
+			margin: 0 auto 115px;
+		}
+
+		@include helpers.desktop() {
+			padding: 0;
+			margin: 0 auto 49px;
+		}
+
+		@media (min-width: 1970px) {
+			width: 1970px;
+		}
+	}
+
+	&__title {
+		margin-bottom: 55px;
+
+		@include helpers.mobile() {
+			margin-bottom: 60px;
+		}
+
+		@include helpers.tablet() {
+			margin-bottom: 65px;
+		}
+
+		@include helpers.desktop() {
+			margin-bottom: 95px;
+		}
+	}
+
+	&__galary {
+		margin-bottom: 30px;
+
+		@include helpers.tablet() {
+			margin-bottom: 40px;
+		}
+
+		@include helpers.desktop() {
+			width: 1170px;
+			margin: 0 auto 60px;
+		}
+	}
+}
+
+.galary {
+
```

Comment:
remove this empty string, check all such places

Reply:
Done.